content/articles/why-you-should-use-wakanda/index.md

```diff
@@ -0,0 +1,63 @@
+### Introduction
+[Wakanda](https://wakanda.github.io/) is an open-source platform which allows the user to easily and quickly create applications that can be utilized as mobile applications and web application using JavaScript. This definition is sweet enough to catch interest but is obvious that developers don’t fall for this line.
+
```

Comment:
Add another line here to complement the line: `This definition is sweet enough to catch interest but is obvious that developers don’t fall for this line.` 

For example, you can say `Let's look at some cool features of Wakanda.` Or something of sort